src/rds/startOneDBInstance.js

```diff
@@ -0,0 +1,16 @@
+const AWS = require('aws-sdk');
+
+module.exports = function startOneDBInstance(arn) {
+  const rds = new AWS.RDS();
+
+  var instanceId = arn.split(':').pop();
+  console.log("Starting "+instanceId+" ...");
+  return new Promise((resolve, reject) => {
```

Comment:
Unnecessary wrapped promise and catch.

Reply:
Updated per most of your comments, except the *One* functions - these need a promise wraparound, as they need to return promise, not arn. I can probably restructure the code to extract arns in startDBInstances, but I like the idea of arn-in arn-out...

Reply:
Agree with your comment of arn-in arn-out, however you can achieve this as follows without the need to wrap in a promise
```
function func(arn) {
 return promiseChain().then(() => arn);
}
```
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Basic_Syntax